lib/dgram.js

```diff
@@ -495,6 +495,19 @@ Socket.prototype.setMulticastLoopback = function(arg) {
 };
 
 
+Socket.prototype.setMulticastInterface = function(interfaceAddress) {
+  this._healthCheck();
+
+  if (!interfaceAddress) {
```

Comment:
I am going to go with the typeof check since String(undefined) would be relying on "undefined" being a (hopefully) unparseable address and obfuscate which type of error was made. (It will also align with the change I've made in the wrapper based on another comment eliminating any window for callers of this function to trigger an abort instead of an exception.)